html/wtest.html

```diff
@@ -1,711 +1,150 @@
-        } else if (showhide === "hide") {
-          document.getElementById("popupbox").style.visibility = "hidden";
-        }
-
-        const forgotPasswordLink = document.createElement("a");
-        forgotPasswordLink.href = "https://account.wire.com/forgot/";
-        forgotPasswordLink.innerText = "Forgot password?";
-        document.getElementById("popupbox").appendChild(forgotPasswordLink);
-      }
-
-      function loginClick() {
-        const username = document.getElementById("username").value;
-        const password = document.getElementById("password").value;
-
-        // Clear login password in form
-        document.getElementById("password").value = "";
-
-        login("hide");
-        backendLogin(username, password);
-      }
-
-      function stopAllCalls() {
-        for (const tcall of callset.tcalls) {
-          tcall_close(tcall);
-        }
-        callset.nconns = 0;
-        callset.nattempts = 0;
-        callset.tcalls = [];
-      }
-
-      function restartClick() {
-        const restartButton = document.getElementById("restartBtn");
-        console.log("restartClick");
-
-        if (callset.is_running) {
-          callset.is_running = false;
-
-          if (statsInterval) {
-            clearInterval(statsInterval);
-          }
-
-          stopAllCalls();
-          restartButton.classList.remove("on");
-        } else {
-          callset.is_running = true;
-          candUl.innerHTML = "";
-          restartButton.classList.add("on");
-          doStart();
-        }
-      }
-    </script>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="../assets/favicon.ico"
+      type="image/x-icon"
+      rel="shortcut icon"
+      sizes="48x48"
+    />
+    <meta
+      property="og:title"
+      content="Wire · The most secure collaboration platform"
+    />
+    <meta
+      property="og:description"
+      name="description"
+      content="Business chats, one-click conference calls and shared documents – all protected with end-to-end encryption. Also available for personal use."
+    />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://app.wire.com" />
+    <meta
+      property="og:image"
+      content="https://wire-docs.wire.com/kalina/wire-icon.png"
+    />
+    <title>Wire · Calling Test Tool · Log In</title>
+    <link rel="stylesheet" type="text/css" href="../styles/style.css" />
   </head>
 
   <body>
-    <center>
-      <div id="popupbox">
-        <!-- <center><h3>Wire Call Tester</h3></center> -->
-        <center>
-          <svg
-            style="overflow: visible; padding: 10px 0px 10px 0px"
-            fill="rgb(52, 56, 59)"
-            viewBox="0 0 57 18"
-            width="57"
-            height="18"
-            class="css-13w01qv"
+    <div class="main-container">
+      <div class="login-form">
+        <svg
```

Comment:
maybe we can move some/all of the svgs to individual files and then import them instead to clean up this file some more.